utils/setup.cfg

```diff
@@ -3,3 +3,16 @@
 # 3. If at all possible, it is good practice to do this. If you cannot, you
 # will need to generate wheels for each Python version that you support.
 universal=1
+
+[nosetests]
+tests=../,../roles/openshift_master_facts/test/,test/
```

Comment:
note: the `-` x `_` failed in one environment but worked in a different one for me. Possibly due to differing versions of deps.